dm/worker/config.go

```diff
@@ -77,6 +77,7 @@ type Config struct {
 	EnableGTID  bool   `toml:"enable-gtid" json:"enable-gtid"`
 	AutoFixGTID bool   `toml:"auto-fix-gtid" json:"auto-fix-gtid"`
 	RelayDir    string `toml:"relay-dir" json:"relay-dir"`
+	MetaDir     string `toml:"meta-dir" json:"meta-dir"`
```

Comment:
Do we need one more option such as `EnableAutoRestore` to control whether to restore tasks after dm-worker restart?

Reply:
why need to control it?

Reply:
For example one task causes DM-worker crash (root cause is DM's bug), and if user restarts the DM-worker, the task auto recovers and DM-worker crashes again.

Reply:
it shows that there'e something wrong with task verification , and it is really an exception handling scenario, we should handle it in other way, may not just ignore

Reply:
I mean the task starts normally, runs for a while, and then meets an unexpected error. The scenario is extreme but once it happens, it is hard to handle. (Maybe user can change the content of dm_worker_meta file)

Reply:
yep, we should implement it in dmctl